gradle.properties

```diff
@@ -61,3 +61,21 @@ securityFallbackApiEndpoint=do_not_change_here
 # Currently, this is only used for assembling the APK where the build process
 # strips debug symbols from the APK.
 androidNdkVersion=21.3.6528147
+
+# Sentry Config
+## DSN
+qaSentryDsn=do_not_change_here
+sandboxSentryDsn=do_not_change_here
+stagingSentryDsn=do_not_change_here
+productionSentryDsn=do_not_change_here
+securitySentryDsn=do_not_change_here
```

Comment:
The DSN is also same for all the builds. We only need one.

Reply:
Oh okay, will change it